src/kanata/linux.rs

```diff
@@ -18,8 +20,12 @@ impl Kanata {
     /// thread.
     pub fn event_loop(kanata: Arc<Mutex<Self>>, tx: Sender<KeyEvent>) -> Result<()> {
         info!("entering the event loop");
-
+    
+        let (preprocess_tx, preprocess_rx) = std::sync::mpsc::sync_channel(100);
         let k = kanata.lock();
+        let debounce_duration = Duration::from_millis(k.linux_debounce_duration);
```

Comment:
Reading and passing this here means live reloads will not update correctly. This should be read within the preprocessor loop.

Likely would be good to have the datatype within the Kanata struct be an `Arc<Mutex<_>>` for fine-grained locking of this value.